server/threads/sql/index.ts

```diff
@@ -9,6 +9,41 @@ const createThread = `
       $/thread_options/)
     RETURNING id`;
 
+    const deleteThread = `
```

Comment:
This makes sense, but since we're at it we could split these more just so it's easier to read (and can be reused). I think what would make sense is:

1) `delete tags`
2) `delete comments`
3) `delete posts`
4) `delete thread`

Feel free to just add it as a TODO!